manifests/ntpd.pp

```diff
@@ -0,0 +1,19 @@
+# Configure NTP daemon
+class foreman_network::ntpd {
+  if ($foreman_network::configure_ntpd) {
+
+    exec { 'NTP sync':
```

Comment:
This is not suited for continuous use. I get that you want a forced sync in the install scenario. Not sure what's the best solution though.